doc/howto/storage_volumes.md

```diff
@@ -131,6 +156,40 @@ All I/O limits only apply to actual block device access.
 Therefore, consider the file system's own overhead when setting limits.
 Access to cached data is not affected by the limit.
 
+````
+```` {group-tab} UI
+To attach a storage volume to an instance, navigate to the configuration page of the target instance.
+
+In the secondary menu, select {guilabel}`Disk` under {guilabel}`Devices`.
+
+From here, select {guilabel}`Attach disk device`.
+
+```{figure} /images/storage/storage_volumes_attach_to_instance_1.png
+:width: 80%
+:alt: Attach a storage volume to an instance - Disk configuration page
+```
+
+The resulting modal will allow you to choose your disk type. To attach a new custom volume, select {guilabel}`Attach custom volume`.
+
+```{figure} /images/storage/storage_volumes_attach_to_instance_2.png
+:width: 80%
+:alt: Attach a storage volume to an instance - Attach disk device modal
+```
+
+Next, you can select a pre-existing volume to attach to the instance by clicking {guilabel}`Select`, or create a new custom volume by selecting {guilabel}`Create volume`.
+
+```{figure} /images/storage/storage_volumes_attach_to_instance_3.png
+:width: 80%
+:alt: Attach a storage volume to an instance - Attach custom volume modal
+```
+
+Once the modal closes, you may need to add a Mount Point file path in the {guilabel}`Mount point` field.
+Finally, you can proceed to either save your instance changes, or create your instance by selecting {guilabel}`Create` (if you were in the instance creation process).
```

Comment:
I think we should focus on instance edit. Then have a new sentence say, this is also possible in creation. The sentence structure here is mixing both information, making it hard to understand.